INFO 2025-09-10 21:21:50,901 [main.py (startup)] BOT STARTED
INFO 2025-09-10 21:22:09,289 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK portfolio
INFO 2025-09-10 21:22:18,437 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK portfolio
INFO 2025-09-10 21:22:20,675 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:22:24,728 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:22:28,154 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:22:29,906 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:22:32,200 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK click_portfolio_skin
INFO 2025-09-10 21:22:42,195 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK delete_messagen
INFO 2025-09-10 21:22:44,219 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK click_portfolio_skin
INFO 2025-09-10 21:22:51,812 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK delete_portfolio_skin
INFO 2025-09-10 21:22:56,320 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK click_portfolio_skin
INFO 2025-09-10 21:23:12,183 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK delete_portfolio_skin
INFO 2025-09-10 21:23:17,098 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:23:19,759 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:24:37,482 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:24:39,024 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:24:42,715 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK click_portfolio_skin
INFO 2025-09-10 21:24:52,123 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK delete_messagen
INFO 2025-09-10 21:25:11,267 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK search
INFO 2025-09-10 21:25:15,195 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER state_user_router CALLBACK skins_search
ERROR 2025-09-10 21:25:15,832 [log.py (__call__)] Separator symbol ':' can not be used in value skin_name='CS:GO Weapon Case 3'
Traceback (most recent call last):
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\bot\bot\middleware\log.py", line 29, in __call__
    return await handler(event, data)
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\bot\bot\middleware\user.py", line 73, in __call__
    return await handler(event, data)
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\venv\lib\site-packages\aiogram_tool\depend\components\inner_middleware.py", line 42, in __call__
    return await handler(event, data)
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\venv\lib\site-packages\aiogram\dispatcher\event\handler.py", line 43, in call
    return await wrapped()
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\bot\bot\handlers\states\user\router.py", line 76, in skins_search
    reply_markup=paginate_buttons(
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\bot\bot\utils\buttons\inline\user\buttons.py", line 49, in paginate_buttons
    ).pack()
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\venv\lib\site-packages\aiogram\filters\callback_data.py", line 101, in pack
    raise ValueError(
ValueError: Separator symbol ':' can not be used in value skin_name='CS:GO Weapon Case 3'
INFO 2025-09-10 21:35:25,979 [main.py (startup)] BOT STARTED
INFO 2025-09-10 21:35:48,492 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK search
INFO 2025-09-10 21:35:52,192 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER state_user_router CALLBACK skins_search
INFO 2025-09-10 21:35:55,746 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_skin_search
INFO 2025-09-10 21:36:12,898 [main.py (shutdown)] BOT SHUTDOWN
INFO 2025-09-10 21:37:57,812 [main.py (startup)] BOT STARTED
INFO 2025-09-10 21:38:02,910 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK search
INFO 2025-09-10 21:38:06,752 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER state_user_router CALLBACK skins_search
INFO 2025-09-10 21:38:09,290 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_skin_search
INFO 2025-09-10 21:38:11,894 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK click_skin_search
INFO 2025-09-10 21:38:15,798 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK portfolio
INFO 2025-09-10 21:38:18,233 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:38:24,460 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK paginate_portfolio_skin
INFO 2025-09-10 21:39:52,705 [main.py (shutdown)] BOT SHUTDOWN
INFO 2025-09-10 21:41:02,374 [main.py (startup)] BOT STARTED
INFO 2025-09-10 21:41:15,800 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK profile
ERROR 2025-09-10 21:41:16,493 [log.py (__call__)] profile() missing 1 required positional argument: 'user'
Traceback (most recent call last):
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\bot\bot\middleware\log.py", line 29, in __call__
    return await handler(event, data)
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\bot\bot\middleware\user.py", line 73, in __call__
    return await handler(event, data)
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\venv\lib\site-packages\aiogram_tool\depend\components\inner_middleware.py", line 42, in __call__
    return await handler(event, data)
  File "C:\Users\dyadh\OneDrive\Рабочий стол\FastFy\venv\lib\site-packages\aiogram\dispatcher\event\handler.py", line 43, in call
    return await wrapped()
TypeError: profile() missing 1 required positional argument: 'user'
INFO 2025-09-10 21:41:27,915 [main.py (shutdown)] BOT SHUTDOWN
INFO 2025-09-10 21:41:34,102 [main.py (startup)] BOT STARTED
INFO 2025-09-10 21:41:37,920 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK profile
INFO 2025-09-10 21:41:50,162 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK profile
INFO 2025-09-10 21:41:58,050 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER user_callback_router CALLBACK change_percent_user
INFO 2025-09-10 21:42:01,938 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER state_user_router CALLBACK user_change_percent
INFO 2025-09-10 21:42:09,106 [log.py (__call__)] USER 2054556183~shayZi12345 ROUTER slash_commands_user_router CALLBACK profile
